addon/components/frost-link.js

```diff
@@ -71,6 +71,11 @@ export default LinkComponent.extend(PropTypeMixin, HookMixin, SpreadMixin, {
     icon: PropTypes.string,
     priority: PropTypes.oneOf(validPriorities),
     routeNames: PropTypes.array,
```

Comment:
You need to check for it being assigned, probably in `didReceiveAttrs` and if it is, call `deprecate` (from `Ember`). If the `routeNames` paran still functions as expected, then that should be all needed. If not, you'll need to take what was given in `routeNames` and convert it to whatever is necessary to achieve the same as what `routeNames` used to do